src/Paket.Core/Versioning/FrameworkHandling.fs

```diff
@@ -658,6 +656,7 @@ type FrameworkIdentifier =
         | DotNetFramework FrameworkVersion.V4_7_1 -> [ DotNetFramework FrameworkVersion.V4_7; DotNetStandard DotNetStandardVersion.V2_0 ]
         | DotNetFramework FrameworkVersion.V4_7_2 -> [ DotNetFramework FrameworkVersion.V4_7_1 ]
         | DotNetFramework FrameworkVersion.V4_8 -> [ DotNetFramework FrameworkVersion.V4_7_2 ]
+        | DotNetFramework FrameworkVersion.V5 -> [ DotNetCoreApp DotNetCoreAppVersion.V3_1; DotNetStandard DotNetStandardVersion.V2_1 ]
```

Comment:
should we add DotNetFramework FrameworkVersion.V4_8 as compat here?

Reply:
This is something that is not clear for now... net5.0 should implement all netcoreapp3.1, but it definitely don't implement all net48 (remoting/messaging/etc...). So I'd say no.

Reply:
thanks! so we keep it as is.